Vizzuality/marxan-cloud
# FE (feat): upload feature by CSV [MRXN23 295]

## Upload feature by CSV

### Overview

This PR adds the feature upload functionality via CSV.
The upload tabs component can be reused in the cost surface section.

### Designs

[Figma design](https://www.figma.com/file/0xD4gR9pCqU7NROnlSW6ck/Marxan-Layout---Redesign?node-id=1710%3A45220&mode=dev)

### Testing instructions

Its recommended to test in the Rwanda 22 project.
From the upload features modal in the Inventory Panel, go to the CSV tab and check that you can download the template correctly.
Afterwards, you can download the attached CSV in the JIRA task and test that you can upload the features, rename, tag and delete them. 
Finally check if it appear in map.

<img width="523" alt="Screenshot 2023-08-16 at 13 32 22" src="https://github.com/Vizzuality/marxan-cloud/assets/51995866/a4cc8d97-5244-4ef7-adef-eb03c2251b24">
<img width="658" alt="Screenshot 2023-08-16 at 13 34 04" src="https://github.com/Vizzuality/marxan-cloud/assets/51995866/2aafa8e6-2a27-4e53-8f55-64861c8e651e">

### Feature relevant tickets

[MRXN23-295](https://vizzuality.atlassian.net/browse/MRXN23-295)


[MRXN23-295]: https://vizzuality.atlassian.net/browse/MRXN23-295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
